gui/components/context_menu.py

```diff
@@ -8,8 +8,23 @@ class ContextMenu(QObject):
     def __init__(self):
         super(ContextMenu, self).__init__('contextMenu_PopupItem')
         self._menu_item = QObject('contextMenuItem')
+        self._context_add_watch_only_option = QObject('contextMenuItem_AddWatchOnly')
+        self._context_delete_account_option = QObject('contextMenuItem_Delete')
+        self._context_edit_account_option = QObject('contextMenuItem_Edit')
 
     @allure.step('Select in context menu')
     def select(self, value: str):
         self._menu_item.real_name['text'] = value
         self._menu_item.click()
+
+    @allure.step('Select add watched address option from context menu')
```

Comment:
Watch only or watch address here? Because method is add watch only and the name of step is the same with the one on 107 row here of https://github.com/status-im/desktop-qa-automation/pull/162/files#diff-5aa70b057231c56128b0c3f7e0c79b42ab8ea3ccd94e7f0132edf831cfa3521e Maybe better to rename it?

Reply:
yes, sure, fixed